LoopKitUI/Views/DismissibleKeyboardTextField.swift

```diff
@@ -16,21 +16,27 @@ public struct DismissibleKeyboardTextField: UIViewRepresentable {
     var textColor: UIColor
     var textAlignment: NSTextAlignment
     var keyboardType: UIKeyboardType
+    var autocapitalizationType: UITextAutocapitalizationType
+    var autocorrectionType: UITextAutocorrectionType
 
     public init(
         text: Binding<String>,
         placeholder: String,
         font: UIFont = .preferredFont(forTextStyle: .body),
         textColor: UIColor = .label,
         textAlignment: NSTextAlignment = .natural,
-        keyboardType: UIKeyboardType = .default
+        keyboardType: UIKeyboardType = .default,
+        autocapitalizationType: UITextAutocapitalizationType = .none,
+        autocorrectionType: UITextAutocorrectionType = .yes
```

Comment:
Sorry, please check the docs here:
- https://developer.apple.com/documentation/uikit/uitextinputtraits/1624447-autocapitalizationtype
- https://developer.apple.com/documentation/uikit/uitextinputtraits/1624453-autocorrectiontype

The default values you want are `.sentences` and `.default`.

Reply:
Thank you! Sorry for getting it wrong the first time...